edwb_app_intro.py

```diff
@@ -340,19 +291,20 @@ def log_helper(action,route):
         elif func_type == 'search_slide':
             resolve_slide(route_ele[beg],route_ele[beg+1],'search_results',slide_name=route_ele[beg+2].replace('%20',' '),log=True,action=action)
         else:
-
-            model.log(request.remote_addr,'',datetime.datetime.now(),action)
+            model.log(request.remote_addr,second_log_block,datetime.datetime.now(),action)
 
 @app.route('/log_action',methods=['GET', 'POST'])
 def log_action():
     request_dict = json.loads(request.data)
     action = request_dict['action']
     route = request_dict['route']
-    log_helper(action,route)
+    #updated slide gets the name of the destination slide
+    updated_slide = request_dict['updated_slide']
```

Comment:
Add default values for action, route, updated_slide variables , set all to "None"

Reply:
Thank you, I set all of action, route, updated_slide variables in log_helper declaration to None.

Reply:
I have set  action, route, updated_slide variables in the log_helper declaration to None